src/main/java/com/espro/flink/consul/jobregistry/ConsulRunningJobsRegistry.java

```diff
@@ -17,9 +22,9 @@
 /**
  * Stores the status of a Flink Job in Consul.
  *
- * @see JobSchedulingStatus
+ * @see JobResultStore
  */
-public final class ConsulRunningJobsRegistry implements RunningJobsRegistry {
+public final class ConsulRunningJobsRegistry implements JobResultStore {
```

Comment:
Let us change the logic here to hold a list of job ids per state (dirty, clean)

Reply:
Currently we are storing for every jobId the status to the key-value store. The new approach due to the API change of Flink would be to store `clean` and `dirty` as keys along with a set of jobIds as values.

Example: `[clean : [], dirty : [jobId1, jobId2]]`

Reply:
done